common/metrics/src/lib.rs

```diff
@@ -357,6 +364,11 @@ impl K8s {
         K8S_LINES.inc();
     }
 
+    pub fn increment_namespace_lines(&mut self, namespace: &str) {
+        // NOTE: Consider bounding the cardinality of namespace labels?
```

Comment:
IME dropping or coalescing labels can be done at the ingest layer and not here. That's a pattern I've seen before as well as one we're using currently and it seems to be accepted.

Reply:
👍 that makes sense and I see enough "unbounded but unlikely to grow exponentially" type labels in production anyways.